azurerm/resource_arm_monitor_log_profile.go

```diff
@@ -152,9 +152,19 @@ func resourceArmLogProfileCreateUpdate(d *schema.ResourceData, meta interface{})
 			duration = d.Timeout(schema.TimeoutUpdate)
 		}
 	}
-	// Wait for Log Profile to become available
-	if err := resource.Retry(duration, retryLogProfilesClientGet(ctx, client, name, meta)); err != nil {
```

Comment:
the linter's complaining about the function `retryLogProfilesClientGet` being unused - could we remove it?

Reply:
Thanks @tombuildsstuff, I've removed the unused func.